action.yml

```
@@ -95,7 +95,9 @@ runs:
       run: |
         # For debug
         # sudo env "PATH=$PATH" uv pip install --system --break-system-packages git+https://github.com/ultralytics/actions@main codespell tomli
-        packages="ultralytics-actions"
+        sudo env "PATH=$PATH" uv pip install --system --break-system-packages git+https://github.com/ultralytics/actions@fix-indent codespell tomli
+        # packages="ultralytics-actions"
+        packages=""
```

❗ **CRITICAL**: Resetting `packages` to an empty string risks `uv pip install $packages` failing with 'no requirements' and disables optional installs appended later. Preserve the current value instead of resetting.

**Suggested change:**
```suggestion
        packages="$packages"
```